kouncil-backend/src/main/java/com/consdata/kouncil/AbstractMessagesController.java

```diff
@@ -67,9 +70,20 @@
     protected List<TopicMessageHeader> mapHeaders(Headers headers) {
         List<TopicMessageHeader> result = new ArrayList<>();
         for (Header header : headers) {
+            String newHeader = null;
+            if (header.value() != null) {
+                ByteBuffer byteBuffer = ByteBuffer.wrap(header.value());
+                if (header.key().equals(KafkaHeaders.ORIGINAL_TIMESTAMP) || header.key().equals(KafkaHeaders.DLT_ORIGINAL_OFFSET)) {
```

Comment:
Nie wiem czy key może być nullem, jeśli tak to warto zabezpieczyć się przed NPE zamieniając kolejność

Reply:
"the header's key; never null" - z docsow

Reply:
No i elegancko :)